_rules/SC2-4-2-page-has-title.md

```diff
@@ -88,6 +88,17 @@ _There are no major accessibility support issues known for this rule._
 </html>
 ```
 
+```html
+<html>
+  <head>
+    <title> <!-- this page has an empty first title --> </title>
+  </head>
+  <body>
+    <title>Title of the page.</title>
+  </body>
+</html>
+```
+
```

Comment:
This test will currently fail the rule, not pass:

> The first title element contains non-empty text.

Reply:
@kasperisager - Good spot. Amended.